www/views/page/content/list.php

```diff
@@ -76,7 +99,10 @@ function cap($text, $len)
     if (!empty($catFilter)) {
         $filtered = $filtered || !(in_array($article->category_id, $catFilter));
     }
-    if (isset($filter)) {
+    if ($role == 1 && $filtercat != 0) {
+        $filtered = $filtered || $article->category_id != $filtercat;
```

Comment:
`->category_id` is `protected`, die moet je eigenlijk even `public` maken als je hem zo wil gebruiken.